public/ga.js

```diff
@@ -0,0 +1,34 @@
+(function() {
+  var _dntStatus = navigator.doNotTrack || navigator.msDoNotTrack;
+  var fxMatch = navigator.userAgent.match(/Firefox\/(\d+)/);
```

Comment:
Why would we even bother with user-agent sniffing? Just check for the `doNotTrack` property and if it's not set, it's not set. We don't need to filter on browser versions, *especially* not when it comes to firefox users.

Also note that `navigator.msDoNotTrack` is only used by IE9 and IE10, neither of which exists anymore. IE11 uses `window.doNotTrack`, and Edge has normal, proper doNotTrack support.